Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:jammy
+
+LABEL Joey Yandle
```

Comment:
```suggestion
MAINTAINER Joey Yandle <joey@trustmachines.co>
```

Reply:
`MAINTAINER` is [deprecated](https://docs.docker.com/reference/dockerfile/#label), but I think this should be changed to
```docker
LABEL org.opencontainers.image.authors="Joey Yandle <joey@trustmachines.co>"
```

Reply:
Fixed @djordon @AshtonStephens